### PR TITLE
Added a new "Development" language that shows the raw translation IDs

### LINKED ISSUE #7094: When development mode is enabled, add a new language that shows raw translation id
The language could be called "Development", or "Raw" or "TranslationId" or something else.

When this language is selected, nothing will be actually translated. Instead the default translation ID will be used see screenshot. This can be useful for Developers, Translators, ... as it is easier to find a translation key than doing a huge reverse search.

Eg developers currently have to use the translation ID to remove or rename a menu item. It can be hard to find the right one to use. This makes it easier that way. It could be also used to quickly check whether one has created translation IDs for everything or whether one forgot something etc. I'm sure there are more advantages.

raw_translations

### PR DESCRIPTION
**Pull request by mnapoli**
Fixes #7094 

This language is shown when development mode is enabled.

### CODE REVIEW

File: tests/PHPUnit/bootstrap.php [hunk 1]
```diff
@@ -88,7 +88,7 @@ function prepareServerVariables()
 
 // General requirement checks & help: a webserver must be running for tests to work if not running UnitTests!
 if (empty($_SERVER['argv']) || !in_array('UnitTests', $_SERVER['argv'])) {
-    checkPiwikSetupForTests();
+//    checkPiwikSetupForTests();
```
Comment (mattab) [inline tests/PHPUnit/bootstrap.php]: little error here - btw maybe we have a problem with this function if you need to comment it?

File: plugins/LanguagesManager/API.php [hunk 2]
```diff
@@ -66,6 +68,13 @@ public function getAvailableLanguages()
             }
         }
 
+        if (!Development::isEnabled()) {
```
Comment (mattab) [inline plugins/LanguagesManager/API.php]: small feedback: this could be in private method

**Comment by mattab** [comment]
Left some feedback, looks good to me and nice use of DI!

**Comment by mnapoli** [comment]
PR updated with feedback.

**Comment by mattab** [comment]
:+1:

**Pull request merged**